styles.css

```diff
@@ -0,0 +1,84 @@
+/* toollbar styles */
+
+.header {
+  background-color: #475466;
+}
+
+.header.webix_toolbar .logo.webix_el_label .webix_el_box {
+  color: white;
+}
+
+.profile-icon.webix_secondary .webix_button {
+  background-color: transparent;
+  color: white;
+}
+
+.profile-icon.webix_secondary .webix_button .webix_icon_btn {
+  color: white;
+}
+
+/* sidebar styles */
+
+.side-bar.webix_view {
+  background-color: #f4f5f9;
+}
+
+.sidebar-item.webix_list_item {
+  border-bottom: none;
+  font-weight: 500;
+}
+
+.sidebar-item.webix_list_item:hover {
+  background-color: #e2e2e3;
+}
+
+.connect-status.webix_view {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #f4f5f9;
+  border-top: none;
+  color: #13d513;
+}
+
+.connect-status.webix_view .webix_template {
+  height: auto;
```

Comment:
Setting CSS sizes for the widgets is a bad idea, it can conflict with sizing calculations in JS
https://docs.webix.com/desktop__styling.html#wherecsscanandcannotbeapplied
Would be better to set a fixed `height` in the JS config

Valid example of usage: styling *custom* content inside a webix view
https://snippet.webix.com/yt5awrlk